### PR TITLE
Add app.gummi.gummi

### PR DESCRIPTION
**Pull request by farribeiro**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

**REWORK** of #3789

### CODE REVIEW

File: app.gummi.gummi.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<component type="desktop">
+  <id>app.gummi.gummi</id>
+  <launchable type="desktop-id">app.gummi.gummi.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>Gummi</name>
+  <summary>The simple LaTeX editor</summary>
+  <description>
+    <p>Gummi is a LaTeX editor written in the C programming language using the GTK+ interface toolkit. It was designed with simplicity and the novice user in mind, but also offers features that speak to the more advanced user.</p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Gummi Team.</p>
```
Comment (barthalion) [inline app.gummi.gummi.appdata.xml:12]: Again, there's absolutely no reason to have that disclaimer here.

Comment (farribeiro) [reply app.gummi.gummi.appdata.xml]: Done https://github.com/flathub/flathub/pull/3832/commits/82bd438f779ec65dd423929a5cd745d2ead83b8e

**Comment by barthalion** [comment]
Have you contacted upstream developer about your submission?

**Comment by farribeiro** [comment]
@alexandervdm

I am submitting your app on flathub, would you kindly comment on this PR

Best regards,
Fábio

**Comment by barthalion** [comment]
bot, build app.gummi.gummi

**Comment by barthalion** [comment]
image
It doesn't seem to detect required tools from the texlive extension.

**Comment by barthalion** [comment]
Ah, wait, I just realized I don't have it installed.

**Comment by farribeiro** [comment]
ty to approval

**Comment by alexandervdm** [comment]
> @alexandervdm
> 
> I am submitting your app on flathub, would you kindly comment on this PR
> 
> Best regards, Fábio

Hello, and thanks for your work! I am unable to verify the package as I don't use flatpak myself, but perhaps someone who requested a flatpak for Gummi some time ago in [this bug](https://github.com/alexandervdm/gummi/issues/127) can help. One note though; Gummi is MIT licensed instead of GPL.

**Pull request closed without merging**